### PR TITLE
Parse asset details from XClarity Client

### PR DESCRIPTION
**Pull request by MaysaMacedo**
This PR is able to parse the following info from XClarity Client:
- contact
- description
- location
- room 
- rack
- lowest rack unit

depends on #[14827](https://github.com/ManageIQ/manageiq/pull/14827)

### CODE REVIEW

File: app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb [hunk 1]
```diff
@@ -124,7 +124,8 @@ def parse_physical_server(node)
             :firmwares => [] # Filled in later conditionally on what's available
           }
         },
-        :asset_details          => parse_asset_details(node)
+        :asset_details          => parse_asset_details(node),
+        :location_led_state    	=> find_loc_led_state(node.leds)
```
Comment (rodneyhbrown7) [inline app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb]: This change was merged into https://github.com/ManageIQ/manageiq-providers-lenovo/pull/48 yesterday.

Comment (MaysaMacedo) [inline app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb]: It came from this merge, I've just fixed an alignment on this line that the rubocop pointed.

**Comment by juliancheal** [comment]
@MaysaMacedo is this data we already get from the current version of the `xclarity` gem or do we need to update the gem version too?

**Comment by MaysaMacedo** [comment]
@juliancheal No need to update it.

**Pull request merged**